docs/conf.py

```diff
@@ -184,5 +184,5 @@
 # -- Extension configuration -------------------------------------------------
 
 def setup(app):
-    app.add_stylesheet('css/style.css')
-    app.add_stylesheet('css/swagger-ui.css')
+    app.add_css_file('css/style.css')
+    app.add_css_file('css/swagger-ui.css')
```

Comment:
Fixes deprecation warning. `.add_stylesheets()` is deprecated since Sphinx v1.8 and will be removed in v4.0.

Reply:
what version does read the docs use?

Reply:
ahhh 3.5